src/components/layout/Header.jsx

```diff
@@ -1,8 +1,9 @@
 import logo from '../../assets/meet.png'
+import { useUser } from '../../context/UserContext'
 import './Header.css'
 
-const Header = ({ user }) => {
-  const { name, color } = user
+const Header = () => {
+  const { name, color } = useUser()
```

Comment:
Looking at your deploy, this component ins't showing the user in your app anymore, btw.